<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImmersedBoundariesNeptune</class>
 <widget class="QWidget" name="ImmersedBoundariesNeptune">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>932</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBoxActivate">
     <property name="title">
      <string>Immersed Boundaries Method</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QCheckBox" name="checkBoxActivate">
        <property name="text">
         <string>Activate Immersed Boundaries Method</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxDim">
        <property name="toolTip">
         <string></string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxObjects">
     <property name="title">
      <string>Solid objects</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTableView" name="tableView">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;CFD and SYSTEM scales coupling of NEPTUNE_CFD and CATHARE2. The zones where the coupling occurs are defined based on the boundary conditions in NEPTUNE_CFD and circuit elements for CATHARE2.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutFSI">
          <item>
           <widget class="QPushButton" name="pushButtonAdd">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Add a conjugate heat transfer.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonDelete">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Remove the last added conjugate heat transfer.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxExplicit">
     <layout class="QHBoxLayout" name="horizontalLayoutExplicit">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Explicit formula for the selected solid object : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonExplicit">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources_pages.qrc">
          <normaloff>:/new/prefix1/data/icons/22x22/edit-find-replace.png</normaloff>:/new/prefix1/data/icons/22x22/edit-find-replace.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxSTL">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>STL File :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditSTLFile"/>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonSTLFile">
        <property name="text">
         <string>Choose a STL file</string>
        </property>
        <property name="icon">
         <iconset resource="../base/resource_base.qrc">
          <normaloff>:/data/icons/22x22/document-open.png</normaloff>:/data/icons/22x22/document-open.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>312</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxMEDCoupling">
     <layout class="QHBoxLayout" name="horizontalLayoutMEDCoupling">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>MED File : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditMEDFile"/>
      </item>
      <item>
       <widget class="QToolButton" name="toolButtonMEDFile">
        <property name="text">
         <string>Choose a MED File</string>
        </property>
        <property name="icon">
         <iconset resource="../base/resource_base.qrc">
          <normaloff>:/data/icons/22x22/document-open.png</normaloff>:/data/icons/22x22/document-open.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../base/resource_base.qrc"/>
  <include location="resources_pages.qrc"/>
 </resources>
 <connections/>
</ui>
